pom.xml

```diff
@@ -86,6 +86,7 @@
     <!-- Modules here must have a corresponding entry in `dependencyManagement > dependencies` block below! -->
     <module>log4j-changelog</module>
     <module>log4j-changelog-maven-plugin</module>
+    <module>log4j-docgen</module>
```

Comment:
See the comment 3 lines above. Aren't we missing a corresponding entry in `dependencyManagement > dependencies` down below?

Reply:
Fixed